### PR TITLE
feat: add @ridedott eslint plugin

### PR DESCRIPTION
**Pull request by oleg-koval**

### CODE REVIEW

File: test/index.spec.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @ridedott/no-single-line-comment-block */
```
Comment (viestat) [inline test/index.spec.ts:1]: this can be overridden in `.eslintrc` for `*.spec.ts` files only :D

Comment (oleg-koval) [reply test/index.spec.ts:1]: if it's in .eslintrc test are failing as it's expected to fail in one of the new tests

Comment (viestat) [reply test/index.spec.ts:1]: I think this should work
```
  - files:
      - test/**.spec.ts
    rules:
      @ridedott/no-single-line-comment-block: off
```

**Comment by viestat** [comment]
Dependency cycle fixed in https://github.com/ridedott/eslint-plugin/pull/171

File: test/index.spec.ts [hunk 2]
```diff
@@ -123,7 +127,7 @@ const getRuleName = (configuredRule: string): string => {
 };
 
 /*
- * Tests
+ * Tests.
  */
```
Comment (merlinnot) [inline test/index.spec.ts:131]: Wait... would this be reported as an error?

Comment (viestat) [reply test/index.spec.ts:131]: yes, unless you configure it to not do it. (which can be done)

Comment (merlinnot) [reply test/index.spec.ts:131]: It is used a lot at Dott, in many repositories. Is it wise to ban it by default?

Comment (viestat) [reply test/index.spec.ts:131]: same case for no magic number rule, it is disabled for tests

**Pull request merged**